### PR TITLE
Enable Scikit by default but make it optional

### PR DESCRIPTION
**Pull request by levkk**
Some older systems may not have the Python version we want (Python 3.7+). We should not deprive them of the ability to run this extension, they will still benefit from XGBoost, LightGBM and Linfa.

### CODE REVIEW

**Comment by montanalow** [comment]
Do we need the same for huggingface?

File: pgml-extension/Cargo.toml [hunk 1]
```diff
@@ -7,13 +7,14 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [features]
-default = ["pg13"]
```
Comment (montanalow) [inline pgml-extension/Cargo.toml:10]: Since this is actually python (including huggingface)? I'd rename this feature to `python`

Comment (levkk) [reply pgml-extension/Cargo.toml:10]: Yeah true that.

File: pgml-extension/src/orm/model.rs [hunk 2]
```diff
@@ -154,6 +154,13 @@ impl Model {
                     _ => todo!(),
                 },
             },
+
+            #[cfg(not(feature = "sklearn"))]
+            Runtime::python => {
+                error!("Python runtime not supported, recompile with `--features scikit`")
```
Comment (montanalow) [inline pgml-extension/src/orm/model.rs:160]: typo in scikit that you'll fix in the rename to `python`

**Pull request merged**